playground/infrastructure/helper.py

```diff
@@ -13,10 +13,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
+import re
 from dataclasses import dataclass
 from typing import List
 
-from api.v1.api_pb2 import SDK_UNSPECIFIED, STATUS_UNSPECIFIED
+from api.v1.api_pb2 import Sdk, SDK_JAVA, SDK_UNSPECIFIED, STATUS_UNSPECIFIED
+
+SUPPORTED_SDK = {'java': SDK_JAVA}
```

Comment:
Why only one supported SDK? To the end of this sprint there will be 3 supported sdk so lets add them all

Reply:
Added.